spec/EventListener/OrderPaymentPaidListenerSpec.php

```diff
@@ -19,9 +19,8 @@ function let(MessageBusInterface $commandBus): void
 
     function it_dispatches_send_invoice_email_command(MessageBusInterface $commandBus): void
     {
-        $commandBus->dispatch(Argument::that(function (SendInvoiceEmail $command): bool {
-            return $command->orderNumber() === '00000001';
-        }))->shouldBeCalled();
+        $command = new SendInvoiceEmail('00000001');
+        $commandBus->dispatch($command)->willReturn(new Envelope($command));
```

Comment:
We still need to assert that, cause we do not use the return value in any way.

```suggestion
        $commandBus->dispatch($command)->shouldBeCalled()->willReturn(new Envelope($command));
```